dopamine_BL21/11/11_stats.jl

```diff
@@ -0,0 +1,140 @@
+#                                  Coef.  Std. Error       t  Pr(>|t|)  Lower 95%  Upper 95%
+# ──────────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                   292.938      10.4741   27.97    <1e-45    272.132   313.743
+# medium: DA                    -56.6264     13.903    -4.07    <1e-04    -84.243   -29.0097
+# bacteria: BL21               -171.053      13.3124  -12.85    <1e-21   -197.497  -144.61
+# medium: DA & bacteria: BL21    53.0266     17.7668    2.98    0.0036     17.735    88.3182
+# ──────────────────────────────────────────────────────────────────────────────────────────
+
+# dopamine vs. buffer, bacteria vs. no bacteria
+# ie. speed diff of N2 w buffer vs. w dopamine
+# p = 0.0036 --> SIGNIFICANT, NOT AS EXPECTED
+
+
+
+# comparing across mediums for CB strain
+lm(@formula(meanspeed ~ medium*bacteria), subset(trackstats, :worm=>ByRow(==("CB"))))
+# meanspeed ~ 1 + medium + bacteria + medium & bacteria
+
+# Coefficients:
+# ─────────────────────────────────────────────────────────────────────────────────────────
+#                                  Coef.  Std. Error      t  Pr(>|t|)  Lower 95%  Upper 95%
+# ─────────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                  247.358       7.30811  33.85    <1e-52   232.842   261.875
+# medium: DA                    -3.18123    10.6408   -0.30    0.7656   -24.3178   17.9554
+# bacteria: BL21               -22.0502     11.8901   -1.85    0.0669   -45.6685    1.56806
+# medium: DA & bacteria: BL21  -99.1684     16.1558   -6.14    <1e-07  -131.26    -67.0768
+# ─────────────────────────────────────────────────────────────────────────────────────────
+
+# dopamine vs. buffer, bacteria vs. no bacteria
+# ie. speed diff of CB w buffer vs. w dopamine
+# p = <1e-07 --> significant, as expected
+
+
+
+# comparing across mediums for MT strain
+lm(@formula(meanspeed ~ medium*bacteria), subset(trackstats, :worm=>ByRow(==("MT"))))
+# meanspeed ~ 1 + medium + bacteria + medium & bacteria
+
+# Coefficients:
+# ────────────────────────────────────────────────────────────────────────────────────────
+#                                 Coef.  Std. Error      t  Pr(>|t|)  Lower 95%  Upper 95%
+# ────────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                  223.865      11.4304  19.59    <1e-33   201.15    246.581
+# medium: DA                   -12.5965     15.9883  -0.79    0.4329   -44.37     19.1769
+# bacteria: BL21               -47.7135     16.3563  -2.92    0.0045   -80.2182  -15.2088
+# medium: DA & bacteria: BL21  -36.7565     22.4289  -1.64    0.1048   -81.3293    7.81618
+# ────────────────────────────────────────────────────────────────────────────────────────
+
+# dopamine vs. buffer, bacteria vs. no bacteria
+# ie. speed diff of MT w buffer vs. w dopamine
+# p = 0.1048 --> NOT SIGNIFICANT, NOT AS EXPECTED
```

Comment:
MT
dopamine vs. buffer, bacteria vs. no bacteria
p = 0.1048 --> NOT SIGNIFICANT, NOT AS EXPECTED